platform-api/DEPLOYMENT-GUIDE.md

```
@@ -0,0 +1,385 @@
+---
+
+## Chapter 5: Local Dev Verification (1 min)
+
+Start the local dev server to confirm everything works before deploying:
+
+```bash
+uv run pywrangler dev
+```
+
+> **What you should see:**
+> ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)